core/src/main/scala/kafka/server/metadata/KRaftMetadataCache.scala

```diff
@@ -267,7 +267,7 @@ class KRaftMetadataCache(val brokerId: Int) extends MetadataCache with Logging w
    *
    * @param topics                        The iterator of topics and their corresponding first partition id to fetch.
    * @param listenerName                  The listener name.
-   * @param firstTopicPartitionStartIndex The start partition index for the first topic
+   * @param topicPartitionStartIndex      The start partition index for the first topic
```

Comment:
Looks like the description here isn't correct, could you help modify it ? Huge thanks :smiley: 
Maybe `The function that return the start partition index for the topic`.